lib/gringotts/gateways/sagepay.ex

```diff
@@ -170,22 +168,43 @@ defmodule Gringotts.Gateways.SagePay do
   end
 
   @doc """
-  Captures a pre-authorized `amount`.
+  Captures a pre-authorized amount.
 
-  `amount` is transferred to the merchant account by sagepay used in the
-  pre-authorization referenced by `payment_id`.
+  amount is transferred to the merchant's account by sagepay used in the
+  pre-authorization referenced by payment_id.
 
   ## Note
 
-  > If there's anything noteworthy about this operation, it comes here.
-  > For example, does the gateway support partial, multiple captures?
+  * Deferred transactions are not sent to the bank for completion until you capture them using the capture instruction. 
+  * You can release only once and only for an amount up to and including the amount of the original Deferred transaction.
 
   ## Example
 
-  > A barebones example using the bindings you've suggested in the `moduledoc`.
+  The following example shows how one would (partially) capture a previously
+  authorized a payment worth 100 by referencing the obtained authorization id.
```

Comment:
100 pounds or 100 £, units are important (this is not 100 pennies, for example).